cartridges/cloudinary_sfra_changes/cartridge/templates/default/product/productDetails.isml

```diff
@@ -5,7 +5,17 @@
         assets.addJs('/js/productDetail.js');
         assets.addCss('/css/product/detail.css');
     </isscript>
-
+    <isset name="cloudinaryConstants" value="${require('*/cartridge/scripts/util/cloudinaryConstants')}" scope="page" />
+    <isif condition="${pdict.cloudinary && pdict.cloudinary.isEnabled && (pdict.cloudinary.galleryEnabled || pdict.cloudinary.isGalleryEnabled)}">
+        <link rel="preload" href="${cloudinaryConstants.CLD_BASE_PATH + '/' + cloudinaryConstants.CLD_VIDEO_RESOURCE_TYPE + '/list/'+ pdict.product.CLDTagName +'.json'}" as="fetch" crossorigin />
+        <link rel="preload" href="${cloudinaryConstants.CLD_BASE_PATH + '/' + cloudinaryConstants.CLD_IMAGE_RESOURCE_TYPE + '/list/'+ pdict.product.CLDTagName +'.json'}" as="fetch" crossorigin />
+    </isif>
```

Comment:
@asad-rafter should the "is gallery enabled" condition include the 360 and 3d?
and in that case, the "require" statement can be entered into the condition statement